### ISSUE TITLE
Dataverse migration - next batch

### ISSUE THREAD

**Issue by pbinkley** [ualbertalib/HydraNorth#526]
John is ready to generate a new batch of transformed metadata for migration. Has identified two predicates but wants to put more thought into them.
Larry is evaluating the display of Dataverse items in plano - he will look at this new migration to determine further work needed for phase 2 to determine whether DC is sufficient for discovery of Dataverse items in HydraNorth.

**Comment by johnhuck** [comment]
Summary of changes in the new record set:

Presence of new elements
dcterms:abstract (already in data dictionary)
georss:box (just added to data dictionary)

Subtracted elements
1 instance of dcterms:description instead of 2
1 instance of dcterms:rights instead of 2

Root element changed from &lt;oai_dc:dc> to &lt;oai_dc:dcterms>

Namespaces
New namespace for georss: xmlns:georss="http://www.georss.org/georss/
I inadvertently left off the xsi: namespace. This shouldn't affect much, but let me know if it does.

Finally, there is still html markup in the dcterms:abstract field and the dcterms:rights field. We decided last week that any html markup would simply be stripped out, but that this would be accomplished by a script just before ingestion into Hydra. I did not normalize space within the content values of these elements so as not to interfere with this script.

The stylesheet can be found at https://github.com/ualbertalib/metadata-transforms/blob/master/oai_dc_to_dcterms.xsl

**Comment by johnhuck** [comment]
@weiweishi Another thing I have noticed in the new batch of data is the presence of multiple dc:relation elements, which is new, and which I think will break something because our generic file model does not allow multiple instances of dcterms:relation. They are mapped from 'related studies', 'other references' and possibly 'related material' in Dataverse and are not necessarily URLs, which is what is expected in our data model.

**Comment by johnhuck** [comment]
There is no other qualifying label on the dc:relation element in the oai_dc, so it's a similar situation to the multiple dc:rights elements, which we solved by combining several instances into one dc:rights element. That would be the obvious path, except it still wouldn't solve the problem that the content in these fields will not be a URL. In fact, judging from the Dataverse fields these originate from, it's unlikely it will be a URL.

**Comment by johnhuck** [comment]
@weiweishi, I have run some XPath queries on the latest data I sent you, and have characterized the results here, including inferring the cardinality of each element: 

https://docs.google.com/spreadsheets/d/1g-K27MmfRelyBTf_ySLBbhLEYCXBV252Goi_R0gYO8E/edit?usp=sharing

I did not find any other new dc: elements other than dc:relation in the records, and no other elements that have multiple instances that are single value in the object model. So there should not be any other problems with the records.

However, I did find that dc:title and dc:identifier are apparently the only mandatory elements (besides the formatted citation for the data in dc:description). The most significant implication for us is that dc:date (=dcterms:created) is not present in every record.
